File: ios/engine/KMEI/SystemKeyboard/KeyboardViewController.m
//
//  KeyboardViewController.m
//  SystemKeyboard
//
//  Created by Serkan Kurt on 20/06/2014.
//  Copyright (c) 2017 SIL International. All rights reserved.
//

#import "KeyboardViewController.h"
#import "KeymanEngine.h"

@interface KeyboardViewController ()

@end

@implementation KeyboardViewController

- (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil {
    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
    if (self) {
        // Perform custom initialization work here
    }
    return self;
}

- (void)updateViewConstraints {
    [super updateViewConstraints];
    // Add custom view sizing constraints here
}

- (void)viewDidLoad {
    [KMManager setApplicationGroupIdentifier:@"group.KMEI"];
    [KMManager sharedInstance].debugPrintingOn = YES;
    [super viewDidLoad];
    // Perform custom UI setup here
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated
}

@end

File: ios/engine/KMEI/SystemKeyboard/KeyboardViewController.swift
//
//  KeyboardViewController.swift
//  SystemKeyboard
//
//  Created by Gabriel Wong on 2017-10-05.
//  Copyright © 2017 SIL International. All rights reserved.
//

import KeymanEngine

class KeyboardViewController: InputViewController {
  override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
    Manager.applicationGroupIdentifier = "group.KMEI"
    super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
  }

  required init?(coder aDecoder: NSCoder) {
    fatalError("init(coder:) has not been implemented")
  }
}
